users/config.rst

```diff
@@ -341,6 +349,99 @@ markerName
     Name of a directory or file in the folder root to be used as
     :ref:`marker-faq`. Default is ".stfolder".
 
+copyOwnershipFromParent
+    On Unix systems, tries to copy file/folder ownership from the parent directory (the directory it's located in).
+    Requires running Syncthing as privileged user, or granting it additional capabilities (e.g. CAP_CHOWN on Linux).
+
+modTimeWindowS
+    Allowed modification timestamp difference when comparing local and remote file for equivalence.
+    To be used on systems that have unstable modification timestamps, that might change after being observed after
+    the last write operation. Used in Android only.
+
+maxConcurrentWrites
+    Maximum number of concurrent write operations while syncing. Defaults to 2. Increasing this might increase or
+    decrease disk performance, depending on the underlying storage.
+
+disableFsync
+    Disables committing file operations to disk before recording them in the database.
+    Can lead to data corruption if disabled.
+
+blockPullOrder
+    Order in which parts of a file are downloaded. This option controls how quickly the different parts of the
+    file spread between the connected devices, at the cost of causing strain on the storage.
+
+    Available options:
+
+    standard (default):
+        The file is split into N equal continuous part sequences, where N is the number of connected
```

Comment:
```suggestion
        The file is split into N equal continuous sequences, where N is the number of connected
```

Reply:
I think it's imporant to leave part here, because file is split into parts, so it's important to know its sequences of parts.

Reply:
Ah, now I get what distinction you want to stress here. I'd use "blocks" then e.g.

```suggestion
        The blocks of the file are split into N equal continuous sequences, where N is the number of connected
```